src/main/java/net/snowflake/client/jdbc/SnowflakeConnectionV1.java

```diff
@@ -242,12 +264,20 @@ public void close() throws SQLException {
           }
         }
       }
+      if (!openStatements.isEmpty()) {
```

Comment:
should we also log about closing statements in `for` above?

Reply:
I am planning to add the log of the statement closure in the `statement.close()` method which will be added in SNOW-1167988